README.rst

```diff
@@ -71,8 +74,11 @@ if you do so, then we'll be able to include it here and spread the word about it
 Installation
 -------------
 
-coala-bears can be installed with ``pip3 install coala-bears``. If you need more
-information about the installation and dependencies, take a look at our
+In order to run a coala-bear you need to install Python. It is recommended, that you install
+Python3 >= 3.3 from `here <https://www.python.org/downloads/>`__.
```

Comment:
i don't really like the ``In order to run a coala-bear``.. try something different, more explicit and correct. because you basically run coala, with bears given as argument.